scaladoc/resources/dotty_res/styles/scalastyle.css

```diff
@@ -988,8 +988,7 @@ footer .socials {
   color: var(--type);
 }
 
-.signature *[t="t"] {
-  /* Types with links */
+.signature *[t="t"] { /* Types with links */
```

Comment:
I would get rid of that; a lot of styling has changed.

Reply:
I don't understand what you want to remove
Could you make a `suggestion` (from github) ?

Reply:
(The only change is a deleted line-return)

Reply:
The change is purely cosmetic (a new line before a comment has been removed). It can’t have any impact.

Reply:
Right. No point in rolling back anything there.